docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -264,6 +267,95 @@ Name | Type | Description
 * logs
 * traces
 
+## Choosing a load balancing strategy
```

Comment:
I'm on the fence if this could become an article or if it belongs in this reference. One could argue that e.g. `spanmetrics` related comments should be in the `spanmetrics` reference page. A fair solution would be to have a task-based doc about scaling and link to it from everywhere.

Reply:
We could have step by step task based docs in addition to the current explanations. Or, one day we could just convert the current explanations to task based docs. But atm I just don't have the time to explain step by step all the possible scenarios in a task based doc.